src/models/log-v4.js

```diff
@@ -90,15 +90,19 @@ async function watchDeploymentAndDisplayLogs (options) {
     commitId,
     knownDeployments,
     quiet,
-    follow,
     redeployDate,
+    exitDeploymentStategy,
   } = options;
 
   // If in quiet mode, we only log start/finished deployment messages
   !quiet && Logger.println('Waiting for deployment to start…');
   const deployment = await waitForDeploymentStart({ ownerId, appId, deploymentId, commitId, knownDeployments });
   Logger.println(colors.bold.blue(`Deployment started (${deployment.uuid})`));
 
+  if (exitDeploymentStategy === 'deploy-start') {
+    return;
+  }
+
```

Comment:
With `--quiet`, the log stream is not started, thus it can't be followed and an `--exit-on never` strategy can't be applied. It's not a problem but the user should be warned about this.

```suggestion

  if (exitDeploymentStategy === 'never' && quiet) {
    Logger.println(colors.bold.yellow('The "never" exit-on strategy is not compatible with the "quiet" mode, it won\'t be applied'));
  }
```